<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mygp</class>
 <widget class="QWidget" name="mygp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mygp</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>61</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>281</width>
        <height>61</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>61</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>331</width>
     <height>701</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>620</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>管理</string>
    </property>
   </widget>
   <widget class="QListWidget" name="gamelist">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>261</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>731</width>
     <height>771</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="widget_3">
    <attribute name="title">
     <string>首页</string>
    </attribute>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>411</width>
       <height>631</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>广告</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>游戏</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>421</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>lol</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>112</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>游戏1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>游戏2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>游戏3</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>直播</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>171</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>开始直播</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>搜房间号</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="listView_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="listView_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>420</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QListView" name="listView_6">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>420</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
